docs/guides/deep_linking/dl_android_overview/dl_android_init_setup.md

```diff
@@ -102,7 +102,23 @@ If there already is an intent-filter for the Android App Link in the main activi
         android:scheme="https" />
 </intent-filter>
 ```
-
+> 📘 **Adding web redirections**
+>
+> If you want to redirect users to web destinations besides your app and have created templates for those destinations, use the `pathPrefix` attribute to specify the template name, as shown in the following example:
```

Comment:
This text should be revised. See the Slack thread

Reply:
Done